testnet/stacks-node/src/nakamoto_node/sign_coordinator.rs

```diff
@@ -272,25 +303,31 @@ impl SignCoordinator {
                     miners_session,
                     signing_round_timeout: config.miner.wait_on_signers.clone(),
                     next_signer_bitvec,
+                    signer_entries: signer_public_keys,
+                    weight_threshold: threshold,
                 };
-                sign_coordinator
-                    .coordinator
-                    .set_aggregate_public_key(Some(aggregate_public_key));
+                if let Some(aggregate_public_key) = aggregate_public_key {
+                    sign_coordinator
+                        .coordinator
+                        .set_aggregate_public_key(Some(aggregate_public_key));
+                }
                 return Ok(sign_coordinator);
             }
         }
-        let party_polynomials = get_signer_commitments(
-            is_mainnet,
-            reward_set_signers.as_slice(),
-            stackerdb_conn,
-            reward_cycle,
-            &aggregate_public_key,
-        )?;
-        if let Err(e) = coordinator
-            .set_key_and_party_polynomials(aggregate_public_key.clone(), party_polynomials)
-        {
-            warn!("Failed to set a valid set of party polynomials"; "error" => %e);
-        };
+        if let Some(aggregate_public_key) = aggregate_public_key {
```

Comment:
Let's just go ahead and strip this code out for now.  Or, if there's test code that uses it, then let's put it into the test module directly.

Reply:
There is test code that could use it (ie `tests/signer/v1`), but until we have an epoch-gated approach, the miner won't ever trigger this code.

Reply:
Removed 👍🏼